packages/palette/src/elements/Select/Select.tsx

```diff
@@ -71,6 +72,36 @@ export const SmallSelect: SFC<SelectProps> = props => {
   )
 }
 
+/**
+ * A mini drop-down select menu
+ */
+export const MiniSelect: SFC<SelectProps> = props => {
+  return (
+    <MiniSelectContainer {...props}>
+      <label>
+        {props.title && (
+          <Sans size="2" display="inline" mr={0.5}>
+            {props.title}:
+          </Sans>
+        )}
+
+        <select
+          value={props.selected}
+          onChange={event =>
+            props.onSelect && props.onSelect(event.target.value)
+          }
+        >
+          {props.options.map(({ value, text }) => (
+            <option value={value} key={value}>
+              {text}
+            </option>
+          ))}
+        </select>
+      </label>
+    </MiniSelectContainer>
+  )
+}
```

Comment:
Can we add a test for this?

Reply:
I was going to but the test for the Select is non existent. Would love to maybe pair with somebody who knows what the plan for this test was.

Reply:
The Select also missing .ios file, not sure why?

Reply:
Likely because this was moved over from Reaction and was never updated

Reply:
Per discussion with @damassi - will add a separate test file for MiniSelect that tests the props handling and selection function.